### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by AngelofMe** [libretro/docs#750] [Ideas]
like this site
https://retropie.org.uk/docs
this is a libretro site
https://docs.libretro.com

**Comment by fpscan** [comment]
@gouchi @bslenul @hunterk Any color idea for current one or should we have a trigger button that converts to darker theme?

**Reply by bslenul** [reply]
I use a Firefox addon called "Dark Reader" and it looks like that:

![image](https://user-images.githubusercontent.com/33353403/180180046-4e2e6152-8e4d-4b74-8f5e-203bf545356f.png)

I personally really like it. A button to trigger it would be better IMO, having the choice is always nice :D

**Reply by AngelofMe** [reply]
RetroArch and Libretro sites have been hurting everyone's eyes and AMOLED screen devices
Pure black is the least harmful to the human eye
Best You can do is white text on a black background.
Since I install Magisk for An Android Device, I know there's a theme called AMOLED (000000 Background and FFFFFF text)
Libretro members upladed videos on YouTube
I can change YouTube Theme to Dark Mode

**Reply by fpscan** [reply]
> RetroArch and Libretro sites have been hurting AMOLED screen devices 

How long do you stay on the page so it burn the screen?

@bslenul Can you see the hex code?

**Reply by bslenul** [reply]
222425 for the background color it seems.

**Reply by fpscan** [reply]
https://github.com/libretro/docs/commit/58aafaf4ba16fdbf5ad25c004ca0616b5563ae6b
